.gitignore

```diff
@@ -5,4 +5,8 @@ contents.xcworkspacedata
 android/.gradle
 android/.project
 android/.settings
+.vs/
 build
+..*
+*~
+nuget.exe
```

Comment:
jfyi: if this is targeting RNW 0.64, we shouldn't need this now that we got rid of the nuget-exe npm package (we'll just use the nuget.exe from the VS install)